tests/portal/test_registration.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from fauxfactory import gen_string
+from insights.ui.locators import insight_page_locator
+from insights.ui.login import LoginUI
+from insights.ui.selenium_utility import SeleniumUtility
+from insights.ui.search import UISearch
+from insights.vm_provisioning import VirtualMachine
+from selenium import webdriver
+
+class Register(unittest.TestCase):
```

Comment:
The class name for any test should have the word Test in it, something like ClientRegisterTestCase or DirectRegisterTest.